README.md

```diff
@@ -0,0 +1,113 @@
+# wmh-analysis
+## Rationale
+This repository contains the R code developped for the research paper on migration patterns of Western Marsh Harriers 
+(*'Contrasting migration strategies between nearby breeding populations of Marsh Harrier* Circus aeruginosus 
+*using the West African-Eurasian flyway'*) by Wouter Vansteelant et al.
+
+In this paper, GPS tracking data were used to study the wintering migration patterns of *Circus aeruginosus* of 3 breeding populations: 
+1. near the Belgian-Dutch border in the provinces East-Flanders (Belgium) and Zeeland (the Netherlands) (4 tracked birds, 
+data collection by [INBO](www.inbo.be/en) as part of the Flemish contribution to the [LifeWatch](www.lifewatch.be) observatory), 
+2. in the province of Groningen (the Netherlands) (2 tracked birds, data collection by 
+[Dutch Montagu´s Harrier Foundation](http://werkgroepgrauwekiekendief.nl/)), and
+3. southern Sweden (4 tracked birds, data collection by [Lund University](https://www.lunduniversity.lu.se/)).
+
+
+## Repo structure
+
+```
+|-- data                <- Data used in this analysis
+    |--maps             <- Maps downloaded by the scripts, to be ignored by git
+|-- reports             <- Output generated by the scripts, to be ignored by git
+    |-- figures         <- Figures generated by the scripts, to be ignored by git
+    |-- tables          <- Tables (.csv) generated by the scripts, to be ignored by git
+|-- src                 <- Scripts generated for this analysis
+|-- gitignore           <- Files and directories to be ignored by git
+|-- LICENSE             <- Repository license
+|-- README.md           <- Description of this repository
+|-- wmh-analysis.Rproj  <- R project file for this repository
+```
+
+
+## Getting started
+### Packages
+In order to run the scripts properly, the installation of the following R-packages is required:
```

Comment:
For other people to start working on the code, wouldn't it be easier to provide the R command to install all of these packages instead of a list they will have to translate to it anyway?

`install.packages(c("car", "circuar", "emmeans", "fossil", "ggplot2", ...)`

Reply:
Now a separate file, will update here later